bindings/python/docs/source/extension_types.rst

```diff
@@ -0,0 +1,40 @@
+Extension Types
+===============
+
+This tutorial is intended as an introduction to working with
+**PyMongoArrow** and its corresponding extension types. The reader is assumed to be familiar with basic
+`PyMongo <https://pymongo.readthedocs.io/en/stable/tutorial.html>`_ and
+`MongoDB <https://docs.mongodb.com>`_ concepts.
+
+Extension types with Arrow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Both extension types, ObjectId and Decimal128, are supported in Arrow. This means that when using
+a PyArrow table, you can use the corresponding PyMongoArrow extension type like so::
+
+        schema = Schema({"_id": ObjectIdType(), "data": Decimal128StringType()})
+        table = find_arrow_all(coll, {}, schema=schema)
+        print(table)
+        >>> pyarrow.Table
+        >>> _id: fixed_size_binary[12]
```

Comment:
These still aren't extension types, since we're not currently casting the response to `ObjectIdType()`.

Reply:
Yup, we can't make them true arrow extension types without implementing pandas extension types. All of that work has been deferred.

Reply:
Can you please update the title of the PR and the ticket to say "not fully supported"?

Reply:
Done.